sdk/storage/Azure.Storage.Blobs/tests/BlobSasBuilderTests.cs

```diff
@@ -213,6 +218,7 @@ public void ToSasQueryParameters_SnapshotIdentityTest()
             Assert.AreEqual(Constants.Sas.Resource.BlobSnapshot, sasQueryParameters.Resource);
             Assert.AreEqual(Permissions, sasQueryParameters.Permissions);
             Assert.AreEqual(signature, sasQueryParameters.Signature);
+            AssertResponseHeaders(constants, sasQueryParameters);
```

Comment:
Why are only the Response headers pulled out into an Assert?  Ideally we'd have a shared assert for everything or do everything inline.

Reply:
There are some differences between the different tests in what they are asserting. These particular parameters are all related in that they are all overriding response headers. I think it would make sense to group like asserts into methods, but didn't go through and do that for all of the existing asserts.